tests/periph/rtc/main.c

```diff
@@ -178,48 +209,52 @@ int main(void)
     print_time("  Setting alarm to ", &time);
     rtc_set_alarm(&time, cb, &rtc_mtx);
 
-    time = (struct tm){0};
+    time = (struct tm){ 0 };
     ms = 0;
 
     /* verify alarm */
     rtc_get_alarm(&time);
     print_time("   Alarm is set to ", &time);
 
-    time = (struct tm){0};
+    time = (struct tm){ 0 };
     ms = 0;
 
     /* clear alarm */
     rtc_clear_alarm();
     if (IS_USED(MODULE_PERIPH_RTC_MS)) {
         rtc_get_time_ms(&time, &ms);
         print_time_ms("  Alarm cleared at ", &time, ms);
-    } else {
+    }
+    else {
         rtc_get_time(&time);
         print_time("  Alarm cleared at ", &time);
     }
 
     /* verify alarm has been cleared */
-    xtimer_sleep(PERIOD);
+    ztimer_sleep(ZTIMER_SEC, PERIOD);
```

Comment:
ZTIMER_SEC might have a jitter ( it also optionally has a rtc backend with might interfere with this test) -> better use MSEC or USEC